README.md

```diff
@@ -4,7 +4,11 @@
 [![Go Report Card](https://goreportcard.com/badge/github.com/Pigmice2733/peregrine-backend)](https://goreportcard.com/report/github.com/Pigmice2733/peregrine-backend)
 [![GitHub](https://img.shields.io/github/license/Pigmice2733/peregrine-backend.svg)](https://github.com/Pigmice2733/peregrine-backend/blob/master/LICENSE.md)
 
-Peregrine is a REST API server written in Go for scouting and analysis of FIRST Robotics competitions.
+Peregrine is a REST API server written in Go for scouting and analysis of FIRST Robotics competitions. 
```

Comment:
```suggestion
Peregrine is a REST API server written in Go for scouting and analysis of FIRST Robotics competitions. 

```

You should add a line break here, to separate the curt technical description from the description of scouting.